packages/oidc-js/src/models/client.ts

```diff
@@ -38,6 +38,7 @@ interface BaseConfigInterface {
     authorizationCode?: string;
     sessionState?: string;
     validateIDToken?: boolean;
+    customParams?: Map<string, string>;
```

Comment:
I think it will be better if we use an object instead of a Map since that would be more developer-friendly? WDYT? 
cc @jeradrutnam, @brionmario

Reply:
+1

I think we can have a dynamic interface here. Something like this.

```ts
interface CustomParamsInterface {
    [ key: string ]: any
}
```

Reply:
I will improve it. But As the values are attached to the URL params, I hope better to have string other than any?
```
interface CustomParamsInterface {
    [ key: string ]: string
}
```